xcube_sh/sentinelhub.py

```diff
@@ -39,6 +38,29 @@
 from .version import version
 
 
+class SentinelOAuth2Session(requests_oauthlib.OAuth2Session):
+    """
+    This class aids fixing an issue with using the SentinelHubStore when distributing across a dask cluster.
```

Comment:
When documenting Python components it is not necessary to say "This class" as it is obvious that it is a class, so just say "Aids fixing an issue...". I'll fix it!